File: app/src/main/java/net/north101/android/ghplayertracker/SharedPrefs.java
package net.north101.android.ghplayertracker;

import org.androidannotations.annotations.sharedpreferences.DefaultBoolean;
import org.androidannotations.annotations.sharedpreferences.SharedPref;

@SharedPref
public interface SharedPrefs {
    @DefaultBoolean(keyRes = R.string.prefs_house_rule_vantage, value = false)
    boolean houseRuleVantage();
}

File: app/src/main/java/net/north101/android/ghplayertracker/SharedPrefs.kt
package net.north101.android.ghplayertracker

import org.androidannotations.annotations.sharedpreferences.DefaultBoolean
import org.androidannotations.annotations.sharedpreferences.SharedPref

@SharedPref
interface SharedPrefs {
    @DefaultBoolean(keyRes = R.string.prefs_house_rule_vantage, value = false)
    fun houseRuleVantage(): Boolean
}
